doc/architecture/logical_model.adoc

```diff
@@ -0,0 +1,6 @@
+= Logical model
+
+Logical models consume sensor-data data and produce sensor-data data.
+
+An example of a logical model is a sensor-fusion model, which fuses the output of multiple sensor models.
+Another use case is the injection of faults after the sensor model.
```

Comment:
"injection of faults" does not seem a good choice of words. Let's see how to replace it!